src/Support/PublicFolderCache.php

```diff
@@ -0,0 +1,166 @@
+<?php
```

Comment:
All docblocks in this file need to state what type of value their taking. They also need a better description of what they are actually doing. The method name is not enough :)

E.g.
```
/**
     * cache.
     *
     * @author Jonas Schwartz <josc@nodes.dk>
     * @param $path
     * @param null $width
     * @param null $height
     * @param $mode
     * @return mixed
     */
```

`width` and `height` shouldn't be `null` as a single value. If it supports `null` then it should be `integer|null`.